### PR TITLE
Export another metric load / number of cores

### PR DESCRIPTION
**Pull request by monicasarbu**
Implements https://github.com/elastic/beats/issues/420

cc @andrewkroh @ruflin

### CODE REVIEW

File: metricbeat/module/system/cpu/cpu.go [hunk 1]
```diff
@@ -100,6 +100,11 @@ func (m *MetricSet) Fetch() (common.MapStr, error) {
 		"5":  loadStat.Load5,
 		"15": loadStat.Load15,
 	}
+	cpuStat["load_normalized"] = common.MapStr{
```
Comment (ruflin) [inline metricbeat/module/system/cpu/cpu.go]: Should we go here with `load.1` and `load.normlized.1` or `load.1.value` and `load.1.normlized`. I think at least both should be in the namespace `load`. Not sure about the naming below it.

**Comment by ruflin** [comment]
I think the fields.yml updates are missing here (which means probably also the system tests will fail).

File: metricbeat/module/system/cpu/cpu.go (hunk 1 above)
Comment (monicasarbu) [inline metricbeat/module/system/cpu/cpu.go]: I agree that they should be under `load`. I suggest `load.avg.1` and `load.normalized.1`.

File: metricbeat/module/doc.go [hunk 2]
```diff
@@ -26,6 +26,7 @@ The goal is to have a similar experience across all metrics.
 List of standardised words and units across all metricsets. On the left are the ones to be used, on the right the options seen in metricsets.
 
 * avg: average
+* norm: normalized
```
Comment (ruflin) [inline metricbeat/module/doc.go]: I think we should sort the list alphabetical

Comment (monicasarbu) [inline metricbeat/module/doc.go]: sorry, I didn't notice that it's sorted.

File: metricbeat/etc/kibana/visualization/System-load.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-  "visState": "{\"title\":\"New Visualization\",\"type\":\"line\",\"params\":{\"shareYAxis\":true,\"addTooltip\":true,\"addLegend\":true,\"showCircles\":true,\"smoothLines\":false,\"interpolate\":\"linear\",\"scale\":\"linear\",\"drawLinesBetweenPoints\":true,\"radiusRatio\":9,\"times\":[],\"addTimeMarker\":false,\"defaultYExtents\":false,\"setYExtents\":false,\"yAxis\":{}},\"aggs\":[{\"id\":\"1\",\"type\":\"avg\",\"schema\":\"metric\",\"params\":{\"field\":\"system.cpu.load.1\",\"customLabel\":\"Load\"}},{\"id\":\"2\",\"type\":\"date_histogram\",\"schema\":\"segment\",\"params\":{\"field\":\"@timestamp\",\"interval\":\"auto\",\"customInterval\":\"2h\",\"min_doc_count\":1,\"extended_bounds\":{}}},{\"id\":\"3\",\"type\":\"terms\",\"schema\":\"group\",\"params\":{\"field\":\"beat.name\",\"size\":5,\"order\":\"desc\",\"orderBy\":\"1\"}}],\"listeners\":{}}",
```
Comment (ruflin) [inline metricbeat/etc/kibana/visualization/System-load.json]: The kibana files are also in the _meta folder now, so changes should be made there.

**Comment by andrewkroh** [comment]
LGTM

**Comment by ruflin** [comment]
LGTM, but it seems to break the system tests for topbeat on windows.

**Comment by monicasarbu** [comment]
Closed in favor of https://github.com/elastic/beats/pull/2101

**Pull request closed without merging**